src/pipecat/services/deepgram/stt_sagemaker.py

```diff
@@ -0,0 +1,447 @@
+#
+# Copyright (c) 2024–2025, Daily
+#
+# SPDX-License-Identifier: BSD 2-Clause License
+#
+
+"""Deepgram speech-to-text service for AWS SageMaker.
```

Comment:
Why did we choose Deepgram in particular for SageMaker-ifying?

Are there a bunch of other pre-built models that users might choose to host on SageMaker?

Is the plan to add a bunch more SageMaker flavors of other Pipecat services? (Follow up: are we worried about the scalability of this approach?)